src/idl_gen_general.cpp

```diff
@@ -659,7 +659,15 @@ static void GenStruct(const LanguageParameters &lang, const Parser &parser,
   //   int o = __offset(offset); return o != 0 ? bb.getType(o + i) : default;
   // }
   GenComment(struct_def.doc_comment, code_ptr, &lang.comment_config);
-  code += std::string("public ") + lang.unsubclassable_decl;
+  code += "public ";
+  if (lang.language == IDLOptions::kCSharp &&
+      struct_def.attributes.Lookup("csharp_partial")) {
```

Comment:
Oh, you mean to avoid writing `"csharp_partial"` here? I don't think that's the biggest deal.